bin/lib/create.js

```diff
@@ -25,9 +25,10 @@ const { CordovaError, events } = require('cordova-common');
 const utils = require('./utils');
 
 function copyJsAndCordovaLib (projectPath, projectName, use_shared) {
-    fs.copySync(path.join(ROOT, 'CordovaLib', 'cordova.js'), path.join(projectPath, 'www/cordova.js'));
-    fs.copySync(path.join(ROOT, 'cordova-js-src'), path.join(projectPath, 'platform_www/cordova-js-src'));
-    fs.copySync(path.join(ROOT, 'CordovaLib', 'cordova.js'), path.join(projectPath, 'platform_www/cordova.js'));
+    const srcCordovaJsPath = path.join(projectPath, 'www/cordova.js');
+    const platformWwwDir = path.join(projectPath, 'platform_www');
+    fs.ensureDirSync(platformWwwDir);
+    fs.copySync(srcCordovaJsPath, path.join(platformWwwDir, 'cordova.js'));
```

Comment:
Since `fs.copySync ` creates missing target directories, the following should suffice:
```suggestion
    fs.copySync(
        path.join(projectPath, 'www/cordova.js'),
        path.join(projectPath, 'platform_www/cordova.js')
    );
```